functions.yml

```diff
@@ -350,7 +367,8 @@ quality_control:
           [Row(stats=[Row(sampleId='NA12878', mean=2.0, stdDev=1.0, min=1.0, max=3.0)])]
       args:
         - name: genotypes
-          doc: The array of genotype structs
+          doc: An array of genotype structs
+      returns: An array of structs where each struct contains ``mean``, ``stDev``, ``min``, and ``max`` of the genotype depths for a sample. If ``sampleId`` is present in a genotype, it will be propagated to the resulting struct as an extra field.
     - name: sample_gq_summary_stats
       doc: Compute per-sample summary statistics about the genotype quality field in an array of genotype structs
```

Comment:
`Compute per-sample summary statistics about the genotype quality field in an array of genotype structs` -> `Computes per-sample summary statistics about the genotype quality field in an array of genotype structs.`

Reply:
Done